crates/bevy_gizmos/src/config.rs

```diff
@@ -13,6 +13,23 @@ use std::{
     ops::{Deref, DerefMut},
 };
 
+/// An enum configuring how line joints will be drawn.
+#[derive(Debug, Default, Copy, Clone, Reflect, PartialEq, Eq, Hash)]
+pub enum GizmoLineJoint {
+    /// Does not draw any line joints.
+    #[default]
+    None,
+    /// Extends both lines at the joining point until they meet in a sharp point.
+    Miter,
+    /// Draws a round corner with the specified resolution between the two lines.
+    ///
+    /// The resolution determines the amount of triangles drawn per joint,
+    /// e.g. `GizmoLineJoint::Round(4)` will draw 4 triangles at each line joint.
+    Round(u32),
+    /// Draws a bevel to connect the ends of both lines.
```

Comment:
Not blocking: Can we maybe explain what is a bevel? TBH, the first time I readed it, as a non native speaker, I quite of didn't understood.